overlord/hookstate/ctlcmd/helpers.go

```diff
@@ -171,6 +181,84 @@ func runServiceCommand(context *hookstate.Context, inst *servicestate.Instructio
 	}
 }
 
+func validateSnapAndCompsNames(names []string, ctxSnap string) ([]string, error) {
+	comps := make([]string, len(names))
+	for i, name := range names {
+		snap, comp := snap.SplitSnapComponentInstanceName(name)
+		// if snap is present it must be the context snap for the moment
+		if snap != "" && snap != ctxSnap {
+			return nil, fmt.Errorf("cannot install snaps using snapctl yet")
```

Comment:
not sure we want to use yet here, as it might never be possible or not in general form

Reply:
"yet" removed